### DISCUSSION TITLE
Is there any IDE (like IDEA) extension for highlighting Jimple IR files?

### DISCUSSION THREAD

**Discussion by XYHyouKa** [soot-oss/SootUp#704] [General]
As the title, sometimes I have to read Jimple IR codes, it would be better if the codes are highlighted.

**Comment by swissiety** [comment]
http://github.com/swissiety/JimpleLSP enables highlighting support for Jimple in IDEs that support LSP Server integrations

If you are using vscode there exists the corresponding JimpleLSP Plugin.

**Reply by XYHyouKa** [reply]
Thanks for your reply! But IntelliJLSP seems not work on Idea 2022.2.1 with an error (javafx.embed.swing.JFXPanel not found)

**Reply by swissiety** [reply]
the plugin for IntelliJ is not maintained  - for reading Jimple I suggest using vscode.
